src/__stories__/poster-card-story.tsx

```diff
@@ -0,0 +1,175 @@
+import * as React from 'react';
+import {
+    IconInvoicePlanFileRegular,
+    skinVars,
+    Circle,
+    Tag,
+    TagType,
+    IconLightningRegular,
+    ResponsiveLayout,
+    Stack,
+    Text2,
+    Inline,
+} from '..';
+import {PosterCard} from '../card';
+
+export default {
+    title: 'Components/Cards/Poster card',
+};
+
+const BACKGROUND_SRC =
+    'https://images.unsplash.com/photo-1622819584099-e04ccb14e8a7?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1932&q=80';
```

Comment:
we commented in a call we're about to change this to be more robust? is there a specific task for that and I can forget about that in the meantime?

Reply:
we have this ticket
https://jira.tid.es/browse/WEB-776